Proposal-EDGI_ML_project.md

```diff
@@ -0,0 +1,165 @@
+### **Student**
+
+Name: Janak Raj Chadha,  Janak
+
+Affiliation: Visvesvaraya National Institute of Technology, Nagpur, India
+
+Electrical and Electronics Engineering, Bachelor of Technology
+
+Location: Nagpur, Maharashtra, India
+
+Indian Standard Time(IST), GMT +05:30
+
+GitHub handle: janakrajchadha
+
+Slack handle: the\_automator
+
+Email: janakrajchadha@gmail.com
+
+Other Contact: Linkedin - [https://goo.gl/ePFmSs](https://goo.gl/ePFmSs)
+
+Twitter - [@the\_automator\_](https://twitter.com/the_automator_)
+
+### **Project Details**
+
+**Project Title** : Identifying and prioritizing important website changes using learning algorithms.
+
+**Project Abstract/Summary** (&lt;20 words): The project goal is to utilise machine learning algorithms to identify changes on government agency websites which are worth reviewing.
+
+**Describe the need your project fills** :
+
+The ignorant and disparaging stand of the new U.S administration on environmental issues like global warming, air pollution etc. has led to panic as federal agencies gradually make the ocean of scientific data on government servers inaccessible to the public. The project will attempt to tackle the aforementioned issue and fill the following needs:
+
+1. Filter out insignificant changes (which may be a major share of all changes) to webpages, and thus, eliminate the need to spend time reviewing useless information.
+2. Direct analysts to important changes on active websites which will help them to efficiently document and analyse relevant information.
+3. Identify missing data and information files and attempt to find archived versions on other websites.
+
+**Describe how your project meets this need** :
+
+The changes worth reviewing will tend to have some characteristic features which can be classified with the help of various classification algorithms. A considerable amount of time will be devoted to the preparation of the training dataset which is an essential task while working on Machine Learning problems.
+
+A variety of Machine Learning models will be trained on the thoroughly labelled dataset. They will also be tested to evaluate the performance by using some part of the data for validation.
+
+The project will follow the following order of tasks to fill the aforementioned needs:
```

Comment:
I wouldn't say I'm especially interested in one more than the other. I would like to work on all the items I have mentioned in my proposal. 
After your review, I realise that my tentative timeline is a bit ambitious and may not work out as planned. However, as I plan to continue contributing to EDGI even after GSoC, the items I want to work on will remain the same with a change in the timeline.